### DISCUSSION TITLE
Override the social card image for a specific page / blog post

### DISCUSSION THREAD

**Discussion by Techbrunch** [squidfunk/mkdocs-material#4439] [Ideas]
It would be nice if it was possible to override the social card image for a specific page / blog post.

Something like:

```
card:
    - ![](card.png)
```

**Comment by squidfunk** [comment]
Thanks for suggesting. I'm working on a new version of social cards which will be released later this year, which will allow much more flexibility, custom layouts, etc. It will also be possible to override social cards, which is what you requested.
